velox/connectors/Connector.h

```diff
@@ -397,6 +397,10 @@ bool unregisterConnector(const std::string& connectorId);
 /// Returns a connector with specified ID. Throws if connector doesn't exist.
 std::shared_ptr<Connector> getConnector(const std::string& connectorId);
 
+/// Returns a map of all (connectorId -> connector) pairs currently registered.
```

Comment:
Can you add a unit test for this? Thanks!

Reply:
+1